tasks/arrays/arrays.js

```diff
@@ -1,57 +1,89 @@
-/**
- * 1. Write a method to reverse a string; Function should return the string 'This is not a string'
- * if a number is passed
-* @param {string}
-*/
-function reverseString(str) {
-}
-
 /**
  * return an array of characters names
+ * @param {Array} chars
+ * @return {Array} - char names
  */
-function getCharactersNames(chars) {
-}
+
+ function getCharactersNames(chars) {
+    let masname = [];
+    for ( let element of chars)  {
+     masname.push(element.name);   
+      }     
+     
+    return masname;
+    }
+//console.log(getCharactersNames([{name:'Vasia'}, {namedsd:'Vasia'}, {name:'Petia'}]));
+
 
 /**
  * print (console.log) ids of all characters
+ * @param {Array} chars
  */
 function printCharacterNames(chars) {
-}
-
+   
+    for ( let element of chars)  {
+      console.log(element.name );   
+      }     
+      
+    }
+   
 /**
  * return an array of non-human characters
+ * @param {Array} chars
+ * @return {Array} - non human characters
  */
 function getNonHumanCharacters(chars) {
+    return chars.filter(el => (el.species !== 'Human'));
 }
 
 /**
  * return info about character named 'Jerry Smith'
+ * @param {Array} chars
+ * @return {Object} - Jerry object
  */
 function getJerryInfo(chars) {
+    return chars.find(el => (el.name == 'Jerry Smith'));
 }
 
 /**
  * check if all characters are human. return true if all, false if not
+ * @param {Array} chars
+ * @return {boolean}
  */
 function isAllHuman(chars) {
+    return chars.includes('Human');  
```

Comment:
This method only checks if Human exists in the whole JSON file, and obviously this is true :). But you need to check if ALL characters are human, so it should be done by every() method.

Reply:
Done by using every() method.